apps/logging/signals.py

```diff
@@ -52,6 +52,18 @@ def handle_app_authorized(sender, request, auth_status, auth_status_code, user,
 
     # Get auth flow dict from session for logging
     auth_flow_dict = get_session_auth_flow_trace(request)
+    crosswalk_log = {}
+
+    try:
+        crosswalk_log = {
+            "id": user.crosswalk.id,
+            "user_hicn_hash": user.crosswalk.user_hicn_hash,
+            "user_mbi_hash": user.crosswalk.user_mbi_hash,
+            "fhir_id": user.crosswalk.fhir_id,
+            "user_id_type": user.crosswalk.user_id_type
+        }
+    except Exception:
+        crosswalk_log = {}
```

Comment:
@nbragdon It may be good to provide Null/None values in the logging related to Splunk searches/reports returning a Null vs. no-value when these fields are used.

```suggestion
        crosswalk_log = {
            "id": None,
            "user_hicn_hash": None,
            "user_mbi_hash": None,
            "fhir_id": None,
            "user_id_type": None
        }
```